client/question/domain/FeedbackObjectFactory.js

```diff
@@ -88,22 +88,24 @@ tie.factory('FeedbackObjectFactory', [
      *
      * @param {string} text String to be inserted in text paragraph.
      */
-    Feedback.prototype.appendTextParagraph = function(text) {
+    Feedback.prototype.appendTextParagraph = function(text, errorLineNumber) {
```

Comment:
I thought we decided not to do this? It seems messy to have every paragraph know about an "error line number". I also don't think that makes sense conceptually because the error line number relates to the entire feedback and not its individual paragraphs.

Can we pass the errorLineNumber to the learner view directive alongside the feedback as a whole (and not as part of the feedback)?

Ditto below.